beetsplug/web/__init__.py

```diff
@@ -310,6 +314,9 @@ def func(lib, opts, args):
                 self.config['port'] = int(args.pop(0))
 
             app.config['lib'] = lib
+            # Normalizes json output
+            app.config['JSONIFY_PRETTYPRINT_REGULAR'] = False
```

Comment:
Oh, oops! I read it as `True` instead of `False`, haha. Sorry about that and I agree with your reasoning 100%.